### ISSUE TITLE
Cannot use farm crafting bench at all...

### ISSUE THREAD

**Issue by sunnythemagnificent** [War3zuk/War3zuk-FarmLife-v3-Enhanced#1]
https://user-images.githubusercontent.com/67884287/172516817-5dfc49c9-a865-465e-9701-26c57aaf68a2.mov

Uhm, here's a movie that basically shows what's going on. My guess is that the mod needs to be updated for the latest version of 7D2D but uh, I don't know. I'm not an expert.

**Comment by War3zuk** [comment]
Nope, When playing with the vanilla game you require the VP modlet that comes with the files.. That enables all the benches.. The mod itself is made to run along side the War3zuk Launcher overhaul which is MY overhaul that ive done since alpha 16.4..

**Comment by War3zuk** [comment]
https://www.nexusmods.com/7daystodie/mods/2108

That is the Nexus Link that has all the Mod changes & info plus other stuff..

This is the Discord link
https://discord.gg/FcDrGvMHeT
